asset-transfer-basic/application-gateway-java/src/main/java/application/java/App.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Running TestApp: 
+// gradle runApp 
+
+package application.java;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.PrivateKey;
+import org.hyperledger.fabric.client.identity.Signer;
+import java.security.cert.CertificateException;
+import java.util.concurrent.TimeUnit;
+import org.hyperledger.fabric.client.*;
+import org.hyperledger.fabric.client.CallOption;
+import org.hyperledger.fabric.client.CommitException;
+import org.hyperledger.fabric.client.identity.*;
+import org.hyperledger.fabric.client.Contract;
+import org.hyperledger.fabric.client.Gateway;
+import org.hyperledger.fabric.client.identity.Identities;
```

Comment:
These imports contain duplicates (explicit imports within packages that are also imported with wildcards). Please just run your IDE's import auto-tidy.

I would prefer to avoid the wildcard imports to people looking through the code can clearly see where each imported type is coming from.

Reply:
yes Mark.

Reply:
removed unwanted imports